src/server/game/Achievements/AchievementMgr.cpp

```diff
@@ -1692,15 +1683,15 @@ bool AchievementMgr::CanUpdateCriteria(AchievementCriteriaEntry const* criteria,
 
 bool AchievementMgr::ConditionsSatisfied(AchievementCriteriaEntry const* criteria) const
 {
-    for (uint32 i = 0; i < MAX_CRITERIA_REQUIREMENTS; ++i)
+    for (auto /* unnamed struct */ AdditionalRequirement : criteria->AdditionalRequirements)
```

Comment:
"auto /* unnamed struct */" some leftovers ?

Reply:
Nope, see where it's declared, there's no named type for it. I put the comment here to please Shauren.

Reply:
I think it's more readable without the comment in the code (some comments fit better in GitHub as they are just part of the PR review)